bigtop_toolchain/bin/puppetize.sh

```diff
@@ -69,7 +69,8 @@ case ${ID}-${VERSION_ID} in
         puppet module install puppetlabs-stdlib --version 4.12.0
         # Enabling the PowerTools and EPEL repositories via Puppet doesn't seem to work in some cases.
         # As a workaround for that, enable the former here in advance of running the Puppet manifests.
-        dnf config-manager --set-enabled crb
+        dnf config-manager --set-enabled crb devel
+        dnf install -y redhat-lsb initscripts
```

Comment:
Package [dependency on initscripts](https://github.com/apache/bigtop/blob/36a3b44905e062e0b1c657dba6bc82dc84accbb4/bigtop-packages/src/rpm/alluxio/SPECS/alluxio.spec#L75) seems to be already exist. How about adding dependency on redhat-lsb too? We can not expect that the puppetize.sh is applied in all deployment.

Reply:
Thanks.  @iwasakims 
I run alluxio smoke test test, hadoop installation error,the log is :
![image](https://github.com/apache/bigtop/assets/31821890/4b873a35-45ba-4b11-bdeb-d033d8938836)
1. hadoop installed need /lib/lsb/init-functions in bigto/puppet:trunk-rockylinux-9 ,But it didn't.
2. /lib/lsb/init-functions  should pass dnf install redhat-lsb and initscripts.
3. I think this is the base environment and should be installed in puppet.  If you don't want to installed it in puppet,can I move  provisioner/utils/smoke-test.sh ,and add rockylinux9 branch ?

Reply:
Thanks for the advice. @iwasakims 
And I will delete “_dnf install -y redhat-lsb initscripts_”。Only increase ”_dnf config-manager --set-enabled crb devel”。
it succeed too，hadoop can also start。
![image](https://github.com/apache/bigtop/assets/31821890/5f095b4e-cc1f-43ba-8d89-c5452d456330)
![image](https://github.com/apache/bigtop/assets/31821890/982d48fc-ad11-4659-a807-ac2bf86109b6)

Reply:
Thanks  @iwasakims @uscxiexing 
The  “dnf install -y redhat-lsb initscripts” code has deleted, it's work fine in my env.